frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/vm-template/source-image.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { testName } from '../../support';
+import { virtualization } from '../../view/virtualization';
+import { addSource } from '../../view/add-source';
+import { ProvisionSource } from '../../../integration-tests-utils/enums/provisionSource';
+
+const TEMPLATE = 'Red Hat Enterprise Linux 6.0+ VM';
+
+describe('test vm template source image', () => {
+  before(() => {
+    cy.login();
+    cy.visit('');
+    cy.createProject(testName);
+  });
+
+  beforeEach(() => {
+    virtualization.templates.visit();
+    cy.deleteResource({
+      kind: 'DataVolume',
+      metadata: { name: 'rhel6.9', namespace: 'kubevirt-os-images' },
```

Comment:
`kubevirt-os-images` is `openshift-virtualization-os-images` in downstream, not sure what's the best way to address it, it at least can be a const on top of the tests, wdyt?

Reply:
added env variable `KUBEVIRT_OS_IMAGES_NS` which is set to `openshift-virtualization-os-images` by default